lib/SPIRV/SPIRVReader.cpp

```diff
@@ -3816,6 +3773,55 @@ SPIRVToLLVM::SPIRVToLLVM(Module *LLVMModule, SPIRVModule *TheSPIRVModule)
   DbgTran.reset(new SPIRVToLLVMDbgTran(TheSPIRVModule, LLVMModule, this));
 }
 
+Type *SPIRVToLLVM::getTypedPtrFromUntypedOperand(SPIRVValue *Val, Type *RetTy) {
```

Comment:
```suggestion
Type *SPIRVToLLVM::getTypedPtrFromUntypedOperand(const SPIRVValue *Val, const Type *RetTy) {
```

My gut tells me both params should be const for a function named like this. Am I right?